lib/net/validations.rb

```diff
@@ -63,6 +63,21 @@ def self.validate_mac(mac)
       false
     end
 
+    def self.multicast_mac?(mac)
+      return false unless validate_mac(mac)
+
+      # Get most signitifcant byte
+      msb = mac.gsub(/[\.\:\-]/, '').slice(0..1).to_i(16)
```

Comment:
Prefer tr over gsub to remove/change characters, so `mac.tr('.:-', '')` will do the same, but quicker.